src/graphs.js

```diff
@@ -0,0 +1,232 @@
+        borderColor: 'rgba(255, 99, 132, 1)',
+        borderWidth: 1
+    }, 
+    {
+        label: 'Community Average Active Minutes',
+        data: makeYs(communityActivity.findCommunityAverage(today, 'minutesActive')),
+        borderColor: 'rgba(54, 162, 235, 1)',
+        borderWidth: 1
+    }]
+    }
+});
+//WATER
+const waterChart = new Chart(waterCanvas, {
+    type: 'line',
+    data: {
+    labels: weekDates,
+    datasets: {
+        label: 'Your Total Ounces',
+        data: mapWeek(weekWater, 'numOunces'),
+        backgroundColor: 'rgba(255, 99, 132, 0.2)',
+        borderColor: 'rgba(255, 99, 132, 1)',
+        borderWidth: 1
+    }, 
+    }
+});
+//SLEEP
+const sleepChart = new Chart(sleepCanvas, {
+    type: 'line',
+    data: {
+    labels: weekDates,
+    datasets: [{
+        label: `Your Hours Slept`,
+        data: weekMap(weekSleep, 'hoursSlept'),
+        backgroundColor: 'rgba(255, 99, 132, 0.2)',
+        borderColor: 'rgba(255, 99, 132, 1)',
+        borderWidth: 1
+    }, 
+    {
+        label: `Your Sleep Quality`,
+        data: weekMap(weekSleep, 'sleepQuality'),
+        backgroundColor: 'rgba(54, 162, 235, 0.2)',
+        borderColor: 'rgba(54, 162, 235, 1)',
+        borderWidth: 1
+    },
+    {
+        type: 'line', 
+        label: `Your All-Time Average Sleep Quality`,
+        data: makeYs(communitySleep.calculateAvgSleepQualPerDay(sleep.userID)),
+        backgroundColor: 'rgba(54, 162, 235, 0.2)',
+        borderColor: 'rgba(54, 162, 235, 1)',
+        borderWidth: 1
+    },
+    {
+        type: 'line', 
+        label: `Your All-Time Average Sleep Quality`,
+        data: makeYs(communitySleep.calculateAvgSleepHrsPerDay(sleep.userID)),
+        backgroundColor: 'rgba(54, 162, 235, 0.2)',
+        borderColor: 'rgba(54, 162, 235, 1)',
+        borderWidth: 1
+    }]
+    }
+});
+// FRIENDS
+const friendChart = new Chart(friendCanvas, {
+    type: 'line',
+    data: {
+    labels: weekDates,
+    datasets: [{
+        label: `Your Steps`,
+        data: mapWeek(weekActivities, 'numSteps'),
+        backgroundColor: 'rgba(255, 99, 132, 0.2)',
+        borderColor: 'rgba(255, 99, 132, 1)',
+        borderWidth: 1
+    }, 
+    {
+        label: `${getFriendName(user.friends[0])}'s Steps`,
+        data: findFriendSteps(user.friends[0]),
+        backgroundColor: 'rgba(54, 162, 235, 0.2)',
+        borderColor: 'rgba(54, 162, 235, 1)',
+        borderWidth: 1
+    }, 
+    {
+        label: `${getFriendName(user.friends[1])}'s Steps`,
+        data: findFriendSteps(user.friends[1]),
+        backgroundColor: 'rgba(54, 162, 235, 0.2)',
+        borderColor: 'rgba(54, 162, 235, 1)',
+        borderWidth: 1
+    }, 
+    {
+        label: `${getFriendName(user.friends[2])}'s Steps`,
+        data: findFriendSteps(user.friends[2]),
+        backgroundColor: 'rgba(54, 162, 235, 0.2)',
+        borderColor: 'rgba(54, 162, 235, 1)',
+        borderWidth: 1
+    }]
+    }
+
```

Comment:
Nice job with these-- and thanks for organizing them to make them easier to read!